src/dia/ElementView.mjs

```diff
@@ -607,24 +611,28 @@ export const ElementView = CellView.extend({
 
     pointerdown: function(evt, x, y) {
 
-        if (this.isPropagationStopped(evt)) return;
```

Comment:
possibly a breaking change?

Reply:
Good question.

The propagation is stopped now in `Paper.mjs` in `pointerdown()` ([here](https://github.com/clientIO/joint/pull/2110/files/a1680f8490115aa3471afde4c3d354254157a402#diff-cad1adaf695b5bd9fb93a5a9d595e003286a15527816ce6ee7d74e014aa68cb4R2170)).

The `cellView.pointerdown()` is not called if the event propagation has been stopped on `event` or `element:magnet:pointerdown`.

*Note: it seemed to be a dead condition anyway (I wasn't able to enter it).*